src/types/request/v5-user.ts

```diff
@@ -20,7 +19,7 @@ export interface CreateSubApiKeyParamsV5 {
   subuid: number;
   note?: string;
   readOnly: 0 | 1;
-  ips?: string[];
+  ips?: string;
```

Comment:
Instead of this, how about we leave it as an array, and we concatenate the array to a comma separated string? `arr.join(',')`

Reply:
Actually consistency with docs, lets take your route.